src/main/kotlin/io/snyk/plugin/snykcode/core/SnykCodeParams.kt

```diff
@@ -19,10 +20,30 @@ class SnykCodeParams private constructor() : DeepCodeParamsBase(
     { getWaitForResultsTimeout() },
     codeRestApi
 ) {
+    private val requestLogging = Logger.getInstance(SCLogger.presentableName + "RequestLogging").isDebugEnabled
```

Comment:
that should be `fun()` (or val with `get()`) cause we need to re-read setting every time it's called out (user can change it after class created).